### PR TITLE
Separation of libhttpd as separate Debian package

### PR DESCRIPTION
**Pull request by smoe**
Working towards a Debian package that would be acceptable for the distribution. Please give it some some critical review. It is a static library only, hence only a  -dev package is provided.

### CODE REVIEW

**Comment by mwarning** [comment]
Does this mean a user has to download and install two packages?
This seem to be a unnecessary burden.

Anyway, there are efforts underway to replace libhttpd, as it is buggy and not maintained anymore.

**Comment by smoe** [comment]
There is no extra dependency for the binary since it is only a static library that is linked against (actually, I have not changed the build script, so it is still the object files that is linked agianst if I recall/observed correctly). Hence, there is no extra dependency for the nodogsplash binary and no difference for your users. However, developers of other packages could install the libhttpd-dev package as a build dependency.

I just saw the Travis CI build failed and cannot chase that up because of a certificat issue that I fail to circumvent.

**Comment by mwarning** [comment]
Hi smoe, I do not really understand the reason for packaging libhttpd. I would not recommend anyone to use it, since it is not maintained anymore and somewhat buggy.

Also, @lynxis is working towards replacing libhttpd by https://www.gnu.org/software/libmicrohttpd/.
See #46. libmicrohttpd is already in the official Debian repository. Depending on the progress, packaging libhttpd might not be necessary.

I would propose to wait a bit and see how far lynxis gets.

**Comment by smoe** [comment]
Hi mwarning, I thank you for your very helpful comment. My pull request is now basically reduced to an integration of your comment with what is stated in debian/README.source , i.e., no additional package. Since the ftpmasters will otherwise request the separate offering of libhttpd, please kindly adopt / further amend that comment we (I) should then upload nodogsplash to Debian if you agree.

**Comment by mwarning** [comment]
Hi smoe, I would like to merge your changes. Please smash them in one or two commits. Otherwise the changes are hard to follow.
If NDS works for you, then we could upload it to Debian.

**Comment by smoe** [comment]
Please merge https://github.com/nodogsplash/nodogsplash/pull/64 instead.
